tests/std/tests/P0220R1_string_view/test.cpp

```diff
@@ -303,6 +304,24 @@ constexpr bool test_case_buffer_constructor() {
     return true;
 }
 
+constexpr bool test_case_contiguous_constructor() {
+#ifdef __cpp_lib_ranges
+    const vector<char> expectedData{'n', 'o', ' ', 'n', 'u', 'l', 'l'};

```

Comment:
I observe that this doesn't test the `constexpr`ness of the constructor because `vector` isn't yet `constexpr`. Would it be better to replace `vector` with `array`, then add a `static_assert` around line 1049?

Reply:
Yes, great suggestion. Also this test code is never run since I guarded it with `__cpp_lib_ranges` which we currently never define. =(